upgrade_test.py

```diff
@@ -327,6 +327,15 @@ def test_rolling_upgrade(self):
 
         # wait for the 20m read workload to finish
         self.verify_stress_thread(read_20m_stress_queue)
+
+        ### sst3 workload (20m)
+        self.log.info('Starting c-s sst3 workload for 20m')
```

Comment:
Why is it here?
IMO, It should be as part of prepare, before we upgrade any node.

Reply:
fixed.